front/src/main/bll/reducers/SeatsReducer/seats-reducer.ts

```diff
@@ -1,30 +1,71 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
-import { Room, Seat } from '../../../dal/api_client/SeatsService'
+import { Room, Seat, SeatResponseWithNewKey } from '../../../dal/api_client/SeatsService'
+
+enum SeatsLoadingStatuses {
+  IDLE = 'IDLE',
+  LOADING = 'LOADING',
+  SUCCESS = 'SUCCESS',
+  ERROR = 'ERROR',
+}
```

Comment:
create please type file for the statuses

Reply:
done